config/locales/fr.yml

```diff
@@ -73,14 +74,18 @@ fr:
       labels:
         destroy: "Supprimer"
     comments:
+      created_at: "Créé le"
+      resource_type: "Type de ressource"
+      author_type: "Profil de l'auteur"
       body: "Corps"
       author: "Auteur"
       title: "Commentaire"
       add: "Ajouter un commentaire"
+      delete: "Supprimer ce commentaire"
+      delete_confirmation: "Êtes-vous sûr de vouloir supprimer ce commentaire ?"
       resource: "Ressource"
-      resource_type: "Type de ressource"
-      author_type: "Profil de l'auteur"
```

Comment:
He didn't, he just reordered them in bed7a64e1347914d27f0a742b1f047f484af0d75 to match the order in the English version. I'm :+1: for the change!